src/features/JoinView/GuestQR.js

```diff
@@ -58,7 +58,7 @@ export default function QRScanner() {
       joinRequest: 'pending',
       isHost: false,
       individualBills: 0,
-      name: 'Tom',
+      name: 'Josh',
```

Comment:
Why are you hard-coded your name? Shouldn't I be `name: nickname || accountHolderName, //Get userName from google`

Reply:
Ah! That was set when we were first testing the guest/host walkthrough.